components/layout/footer.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <footer class="bg-white-500" aria-labelledby="footer-heading">
+  <footer
```

Comment:
why did you change the footer?

Reply:
I added a border-top to the footer to separate it from the rest of the page